Sources/KituraNet/HTTP/HTTPRequestHandler.swift

```diff
@@ -93,7 +93,7 @@ internal class HTTPRequestHandler: ChannelInboundHandler, RemovableChannelHandle
                     } catch {
                         Log.error("Failed to send error response")
                     }
-                    context.close()
+                    _ = context.close()
```

Comment:
if you use `context.close(promise: nil)`, then that's more efficient and doesn't return anything so you don't need the `_ =`